packages/core/src/js/replay/browserReplay.ts

```diff
@@ -0,0 +1,11 @@
+import { replayIntegration } from '@sentry/react';
+
+const browserReplayIntegration = (options: Parameters<typeof replayIntegration>[0] = {}) => {
+  return replayIntegration({
+    ...options,
+    mask: ['div.sentry-react-native-mask', ...(options.mask || [])],
+    unmask: ['div.sentry-react-native-unmask:not(div.sentry-react-native-mask *) > *', ...(options.unmask || [])],
```

Comment:
super-tiny-l, but I'd say the `div.` part is not really necessary here? 🤔 But just a thought, feel free to disregard.